app/Modules/ExpertPanel/Models/ExpertPanel.php

```diff
@@ -199,37 +196,6 @@ public function evidenceSummaries()
         return $this->hasMany(EvidenceSummary::class);
     }
 
-    /**
-     * @return \Illuminate\Database\Eloquent\Relations\HasOne
-     */
-    public function leadershipAttestation()
-    {
-        return $this->hasOne(\App\Modules\ExpertPanels\Models\LeadershipAttestation::class);
-    }
-
-    /**
-     * @return \Illuminate\Database\Eloquent\Relations\HasOne
-     */
-    public function biocuratorOnboardingAttestation()
-    {
-        return $this->hasOne(\App\Modules\ExpertPanels\Models\BiocuratorOnboardingAttestation::class);
-    }
-
-    /**
-     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
-     */
```

Comment:
Not referenced.  Also an relation with the correct spelling is defined elsewhere in the model.